frontend/ui/uimanager.lua

```diff
@@ -68,7 +67,7 @@ function UIManager:init()
         self:nextTick(function()
             Device:saveSettings()
             Device:powerOff()
-            self:quit(Device:isKobo() and 88)
+            self:quit(Device:isKobo() and 88 or nil)
```

Comment:
Oh, right, good point, devcaps eval to true or false, not true or nil ;).

Reply:
In this special case I would prefer to write:
```
if Device:isKobo() then
    self:quit(88)
else
    self.quit()
end
```